apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerFixtureBase.cs

```diff
@@ -16,25 +16,24 @@
 using Google.Cloud.Spanner.Common.V1;
 using Google.Cloud.Spanner.V1.Internal.Logging;
 
-namespace Google.Cloud.Spanner.Data.CommonTesting
+namespace Google.Cloud.Spanner.Data.CommonTesting;
+
+/// <summary>
+/// Base classes for test fixtures for Spanner.
+/// </summary>
+public abstract class SpannerFixtureBase : CloudProjectFixtureBase
```

Comment:
I think this one and SpannerFixtureBasePostgresSql can be further abstracted using generics on the Database type.